__tests__/unit/octopus-cli-commandline-generation.test.ts

```diff
@@ -83,16 +83,68 @@ test('all the parameters where env has the values', () => {
   ])
 })
 
-test('fails if the overwriteMode is invalid', () => {
+test('fails if the overwriteMode is invalid', async () => {
   const i = makeInputParameters({
     packages: ['test.1.2.3.zip'],
     apiKey: 'API-FOOBAR',
     overwriteMode: 'Banana',
     server: 'https://octopus.server',
     space: 'Spaces-1'
   })
-
-  expect(() => generateLaunchConfig({ parameters: i, env: {} }, console)).toThrow(
+  await expect(generateLaunchConfig({ parameters: i, env: {} }, console)).rejects.toThrowError(
     'The input value, overwrite_mode is invalid; accept values are "FailIfExists", "OverwriteExisting", and "IgnoreIfExists".'
   )
 })
+
+test('globbing packages for matched ending with .zip', async () => {
+  const i = makeInputParameters({
+    packages: ['__tests__/testpkgs/**/*.zip'],
+    apiKey: 'API-FOOBAR',
+    server: 'https://octopus.server'
+  })
+  const env = {
+    OCTOPUS_API_KEY: 'API FOOBAR',
+    OCTOPUS_HOST: 'http://octopusServer',
+    OCTOPUS_SPACE: 'Spaces-1'
+  }
+
+  const launchInfo = await generateLaunchConfig({ parameters: i, env }, console)
```

Comment:
This all looks good; just a thought - might it be useful to pass a working directory / base directory here, so it's explicit as to where it is searching for packages?